src/d3a/d3a_core/sim_results/file_export_endpoints.py

```diff
@@ -211,3 +216,68 @@ def _specific_row(self, slot, market):
                     self.area.strategy.panel_count
                     ]
         return []
+
+
+class KPI:
+    def __init__(self):
+        self.performance_index = dict()
+        self.cep_device = list()
+        self.cep_energy = 0
+        self.total_energy = 0
+
+    def accumulated_trade_energy(self, area):
```

Comment:
From what I understand, this function accumulates the sold energy from the commercial producer and the bought energy from the loads, but the naming does not reflect it ta all. Why not having 2 separate functions that handle these 2 separate tasks, with better names for each? There is no significant performance disadvantage either, since this function is not recursive (recursion is handled by its caller, not from this function).